sklearn/linear_model/tests/test_logistic.py

```diff
@@ -245,20 +247,20 @@ def test_consistency_path():
     f = ignore_warnings
     # can't test with fit_intercept=True since LIBLINEAR
     # penalizes the intercept
-    for method in ('lbfgs', 'newton-cg', 'liblinear'):
+    for method in ('lbfgs', 'newton-cg', 'liblinear', 'sag'):
         coefs, Cs = f(logistic_regression_path)(
-            X, y, Cs=Cs, fit_intercept=False, tol=1e-16, solver=method)
```

Comment:
you're changing the default tol params.

is that on purpose?

@MechCoder any idea?